pocld/daemon.cc

```diff
@@ -66,6 +70,10 @@
     }                                                                          \
   } while (0)
 
+#ifdef ENABLE_REMOTE_ADVERTISEMENT_DHT
+std::thread DHTThread;
```

Comment:
should be a member of the PoclDaemon class, not a global.

Reply:
I have resolved this using a friend function for now. Not sure if that's the 'correct' way to do it.